packages/components/ContextualMenu/src/SubmenuItem.types.ts

```diff
@@ -8,7 +8,10 @@ import { IconProps } from '@fluentui-react-native/icon';
 export const submenuItemName = 'submenuItem';
 export type SubmenuItemTokens = ContextualMenuItemTokens;
 export type SubmenuItemProps = ContextualMenuItemProps;
-export type SubmenuItemState = ContextualMenuItemState;
+
+export interface SubmenuItemState extends ContextualMenuItemState {
+  submenuItemHovered?: boolean;
```

Comment:
naming nit: "is" in front of this variable would make it clearer that this is a boolean.

Reply:
I think other states omit the "is" i.e. focused, hovered, disabled etc